pkg/tagger/utils/expiry.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package utils
+
+import (
+	"errors"
+	"sync"
+	"time"
+)
+
+// Expire regularly pools the source keeping the elements reporting.
+// Currently only used for ECSCollector and active containers.
+// It keeps an internal state to only send the updated elements (only used for containers to start with).
+type Expire struct {
+	sync.Mutex
+	expiryDuration time.Duration
+	lastSeen       map[string]time.Time
+}
+
+// NewExpire creates a new Expire object. Called when a Collector is started.
+// Only used for the ECS collector to start with.
+func NewExpire(expiryDuration time.Duration) (*Expire, error) {
+	if expiryDuration <= 0 {
```

Comment:
let's test `expiryDuration.Seconds()` instead, `time.Duration` is not guaranteed to be comparable to a number (it currently just is [a type alias to int64](https://golang.org/src/time/time.go?s=21095:21114#L610), but that could change

Reply:
👌 
Tested, it works well.
Changed to `expiryDuration.Seconds() <= 0.0` to compare 2 float64